DeviceTests/Caboodle.DeviceTests.Shared/PhoneDialer_Tests.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.Caboodle;
+using Xunit;
+
+namespace Caboodle.DeviceTests
+{
+    public class PhoneDialer_Tests
+    {
+    }
+}
```

Comment:
Is this file necessary?

Reply:
I am going to add some Internal tests as soon as I can pull that up